purchase_order_type/views/view_purchase_order_type.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_purchase_order_type_tree" model="ir.ui.view">
+        <field name="model">purchase.order.type</field>
+        <field name="arch" type="xml">
+            <tree name="Purchase Order Types" decoration-muted="active == False">
+                <field name="sequence" widget="handle" />
+                <field name="active" invisible="True" />
```

Comment:
This column is actually show. See https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0 for more details, but code should be modified like this (not tested):
```suggestion
                <field name="active" column_invisible="True" />
```